lxd/storage/drivers/driver_ceph.go

```diff
@@ -246,7 +250,11 @@ func (d *ceph) Create() error {
 // Delete removes the storage pool from the storage device.
 func (d *ceph) Delete(op *operations.Operation) error {
 	// Test if the pool exists.
-	poolExists := d.osdPoolExists()
+	poolExists, err := d.osdPoolExists()
+	if err != nil {
+		return err
```

Comment:
And we should wrap error with context here too.

More generally we should wrap context when the error won't make much sense to the user without it.